frontend/src/App.tsx

```diff
@@ -65,6 +65,8 @@ type CoveyAppUpdate =
         gameInfo: GameInfo;
         toggleQuit: boolean;
         quitGame: () => void;
+        showLeaderboard: boolean;
```

Comment:
is `showLeaderboard` not already in App State?

Reply:
uhhh i wrote this while troubleshooting hard refresh so i still need to test removing it. added it here so that a boolean value gets populated upon 'doConnect'. i think when you implemented showLeaderboard you made a new default state, so this shouldn't be necessary, but i added it just in case. will take a look asap